Objects/Converters/ConverterRevit/ConverterRevitShared/Extensions/ParameterExtensions.cs

```diff
@@ -49,7 +49,9 @@ internal static class ParameterExtensions
       case StorageType.String:
         return parameter.AsString();
       case StorageType.ElementId:
-        return parameter.AsElementId().ToString();
+        ElementId id = parameter.AsElementId();
+        Element el = parameter.Element.Document.GetElement(id);
```

Comment:
I assume that `parameter.Element` isn't actually an `Element` type (and therefore the same element we're trying to get by `GetElementId`?)

Reply:
It is of "Element" type. According to the documentation, the Element prop is the element that the parameter belongs to. So, if we're converting wall parameters, then the element prop of all the parameters will be same wall object, so we do need to still query the document for the element that the parameter points to.

Reply:
Gotcha, perfect